### ISSUE TITLE
No default Gulp processor

### ISSUE THREAD

**Issue by bitbager** [Sylius/Sylius#10332]
In order to process assets in my local environment/theme, I need to create a new Gulp processor over and over again. I think it'd be nice if there's a default one provided.

**Comment by migmolrod** [comment]
Is this open to suggestions? I see a couple improvements that could be made to this to have a better out-of-the-box experience and following the other Sylius gulp files (Admin and Ui bundles). But may be a tad overkill for most use cases, I don't know.

**Comment by bitbager** [comment]
Of course. Feel free to submit any but keep in my opinion this should be kept rather simple. If the Gulp processing gets too complicated like it is in the current vendor, it'll lead to customization issues for less front-end experienced developers.

**Comment by migmolrod** [comment]
One quick fix would be to add this:
```js
gulp.watch(paths.custom.scss, buildCustomCss);
```
anywhere between the lines 66 and 68 in your [`private/gulpfile.babel.js`](https://github.com/Sylius/Sylius/blob/64a91a679489fb4df28b85e897eee66f701b7259/private/gulpfile.babel.js). As it is now, if you run `yarn watch`, any changes inside the scss folder will be plain ignored.

I also actually renamed the `private` folder to `assets`, which is the recommended in the Symfony Best Practices article (but that's a minor point I guess: the important thing is to have all assets in one folder).

The other change I'd make, I admit, can be overkill. But taking your pull-request as a base, I ended up making my own `private/gulpfile.babel.js` more similar to `vendor/sylius/sylius/src/Sylius/Bundle/AdminBundle/gulpfile.babel.js`. So I just have an `app.js` as entry point that can then 'import' all other js files as needed.
Again, I admit that is quite overkill and, most probably, unnecessary in 99% of the use cases.

For now, adding the bug fix about watching the scss folder would be ok.
